doc/classes/Camera2D.xml

```diff
@@ -4,7 +4,10 @@
 		Camera node for 2D scenes.
 	</brief_description>
 	<description>
-		Camera node for 2D scenes. It forces the screen (current layer) to scroll following this node. This makes it easier (and faster) to program scrollable scenes than manually changing the position of [CanvasItem] based nodes.
+		To use this, you typically embed a [code]Camera2D[/code] node inside any node you want your view to follow (e.g. [code]/YourScene/YourPlayer/Camera2D[/code]). Then enable the property named "current" in your [code]Camera2D[/code] node and you're set.
```

Comment:
I don't think the description should start with "To use this". The description should explain what Camera2D is. Usage instructions should come at the end.